### DISCUSSION TITLE
Approving and commenting PRs

### DISCUSSION THREAD

**Discussion by guerrerotook** [banco-alimentar/alimentestaideia.pt#150] [General]
Now that the development team of banco-alimentar is growing I thing is a good idea to have discussions around the PRs. The main idea is to discuss the code and make sure that we commit quality code to the repo, for that when you're assigned as a reviewer make sure that you comment, give feedback and if everything is ok approve. Never merge the PR, let the author to merge when he feels is the right time.

**Comment by tiagonmas** [comment]
Agree with that best practice @guerrerotook. what's your thoughts @joalmeid @FabioFerreira-LinkConsulting ?

**Comment by joalmeid** [comment]
Makes total sense, specially because time investment will change depending on availability. This way we can raize code quality and shared knowledge!
Reviewers should review, while everyone else may quickly recap work done!
